contracts/Agreement.sol

```diff
@@ -71,8 +64,7 @@ contract Agreement is AccessControl, Ownable {
       return optedOutAccounts;
     }
 
-    function getAddressStatus(address _address) public view returns (string memory) {
-      require(isMember(msg.sender), "Restricted to members.");
+    function getAddressStatus(address _address) public view onlyRole("OG") returns (string memory) {
```

Comment:
Why is this `onlyRole("OG") ?

I don't see any good reason to restrict it.

Reply:
I was converting all of the `require(isMember(...` to `onlyRole` per issue #9 

Should any address be able to check the OG Membership status? Or should only OG DAO members be able to validate that?

Reply:
I don't see any reason to restrict it. I think you can safely remove this check.

Reply:
Adjustment made